cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,66 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
+import SettingsPageObject from '../support/pages/settings.pageObject';
+import HomePageObject from '../support/pages/home.pageObject';
+import SignUpPageObject from '../support/pages/signup.pageObject';
+
 describe('Settings page', () => {
+  let signUpPage;
+  let homePage;
+  let settingsPage;
+  let user;
+
   before(() => {
+    cy.task('db:clear');
 
+    signUpPage = new SignUpPageObject();
+    homePage = new HomePageObject();
+    settingsPage = new SettingsPageObject();
   });
 
   beforeEach(() => {
+    cy.task('db:clear');
 
+    signUpPage.visit();
+    user = signUpPage.registerUser();
+    homePage.assertHeaderContainUsername(user.username);
+    signUpPage.verifyRegistrationSuccess(); // Перевірка успішної реєстрації
   });
 
   it('should provide an ability to update username', () => {
-
+    const newUsername = 'UpdatedUsername'; // Нове ім'я користувача
+    settingsPage.navigateToSettingsPage(); // Перейти на сторінку налаштувань
+    settingsPage.updateUsername(newUsername); // Оновити ім'я користувача
```

Comment:
Ensure that the `updateUsername` method in `SettingsPageObject` is implemented correctly and updates the username as expected.